sandbox_store.go

```diff
@@ -198,7 +201,7 @@ func (c *controller) sandboxCleanup() {
 			dbExists:    true,
 		}
 
```

Comment:
@aboch  let's do
````
// restore sb.config
 sb.processOptions(option...)
 sb.restorePath()
````
here
 instead of at https://github.com/docker/libnetwork/pull/1244/files#diff-ffc9c06e306285f214919ed203907012R707 
because `sb.Key()` relay on `sb.config.useDefaultSandBox`
I had this change on local but forget to push to github in time :)